dist/booking.js

```diff
@@ -16,24 +16,32 @@ const classesByDate = require('../data/classes.json');
 const classesByDay = require('../data/classesByDay.json');
 const { log } = require('./utils/logger');
 
-const filterToBook = (classes, getClassDate) => lessons => {
+const filterToBook = (classes, getClassDate) => (lessons, tomorrowOnly) => {
   let classesToBook = Object.keys(classes)
-    .filter(
-      key =>
-        // Is the class minus 1 days at 7am same or before current date / time
-        getClassDate(key)
+    .filter(key => {
+      if (tomorrowOnly) {
+        // Is the class tomorrow
+        return getClassDate(key)
           .subtract(1, 'day')
-          .hour(7)
-          .minute(0)
-          .second(0)
-          .isSameOrBefore(moment()) &&
-        // Is the class after current date / time
-        getClassDate(key)
-          .hour(23)
-          .minute(59)
-          .second(59)
-          .isSameOrAfter(moment())
-    )
+          .isSame(moment(), 'day');
+      } else {
+        return (
+          // Is the class minus 1 days at 7am same or before current date / time
+          getClassDate(key)
+            .subtract(1, 'day')
+            .hour(7)
+            .minute(0)
+            .second(0)
+            .isSameOrBefore(moment()) &&
+          // Is the class after current date / time
```

Comment:
as the key is always either a day (monday, tuesday, etc) or a date (21/12/2019), I'm not sure why we're bothering with hours minutes and seconds here, maybe I'm missing something.  class is either today or tomorrow...